.tx/config

```diff
@@ -4,6 +4,7 @@ host = https://www.transifex.com
 [qbittorrent.qbittorrent_master]
 file_filter = src/lang/qbittorrent_<lang>.ts
 lang_map = pt: pt_PT
+lang_map = zh: zh_CN
```

Comment:
I suppose you tested it right? Because the document uses comma separated pairs:
```c++
lang_map = pt: pt_PT, zh: zh_CN
```
Just double checking.